app/inputs/select_with_help_input.rb

```diff
@@ -1,3 +1,2 @@
 class SelectWithHelpInput < SimpleForm::Inputs::CollectionSelectInput
-  include WithHelpIcon
```

Comment:
Probably not. I am currently investigating what views in CC and Sufia need to be altered when we remove these classes.